grammars/less.cson

```diff
@@ -273,12 +273,12 @@
     "name": "keyword.other.important.css"
   }
   {
-    'match': '\\$|%|&|\\*|\\/|\\-\\-|\\-|\\+\\+|\\+|~|===|==|=|!=|!==|<=|>=|<<=|>>=|>>>=|<>|<|>|!|&&|\\|\\||\\?\\:|\\*=|(?<!\\()/=|%=|\\+=|\\-=|&=|\\^=|\\/\\b'
+    'match': '\\$|&|\\*|\\/|\\-|\\+|~|=|<=|>=|<|>|&&|\\?\\:|(?<!\\()/=|\\/\\b'
```

Comment:
I was actually wrong: multiple `&`s are supported.  I misread the regex.  I've updated the regex on master.